Orange/widgets/regression/owsgdregression.py

```diff
@@ -44,6 +45,11 @@ class OWSGDRegression(OWProvidesLearner, widget.OWWidget):
     want_main_area = False
     resizing_enabled = False
 
+    loss_function_options = ["Squared loss","Huber","Epsilon insensitive",
+                            "Squared Epsilon insensitive"]
+    penalty_options = ["Absolute norm (L1)","Euclidean norm (L2)",
+                       "Elastic Net (L1 and L2)"]
+    learning_rate_options = ["Inverse scaling","Constant"]
```

Comment:
@kernc 
1. By one per line do you mean
L1
L2 
and so on?
2. By Making them uppercase, u mean change loss_function_options to LOSS_FUNCTION_OPTIONS?